src/ai/backend/common/defs.py

```diff
@@ -44,3 +44,5 @@
 
 DEFAULT_VFOLDER_PERMISSION_MODE: Final[int] = 0o755
 VFOLDER_GROUP_PERMISSION_MODE: Final[int] = 0o775
+
+DEFAULT_DOMAIN_NAME: Final[str] = "default"
```

Comment:
I think it would be great if the default domain name was set to a less commonly used name that's not as user-friendly.

Reply:
we already use this as a default value. Change the default name to another should be a separate epic